### PR TITLE
Use standard comment headers

### PR DESCRIPTION
**Pull request by rgrinberg**
Documented here:

https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html

With this chance, I'm able to use the package directly using
https://github.com/quelpa/quelpa/ for example.

### CODE REVIEW

File: opam-mode.el [hunk 1]
```diff
@@ -1,3 +1,12 @@
+;;; opam-mode.el --- mode for opam files
+
+;; Author: Julien Sagot <ju.sagot@gmail.com>
+;; Version: 0.1
+;; Keywords: opam, ocaml
+;; URL: https://github.com/sagotch/opam-mode.el
```
Comment (sagotch) [inline opam-mode.el]: Should not it be `Homepage`?

**Comment by sagotch** [comment]
Nice, thanks. I was not aware of this convention (or maybe I just forgot them). But I see no reference to a URL field, are you sure this is what you need?

**Comment by rgrinberg** [comment]
Ah yes, that should be `Homepage`.

**Comment by sagotch** [comment]
Sure. I'll have a look at this file which *may* be outdated (I have no idea right now) and at how to submit a package to melpa.

I'll let you know.

**Pull request merged**